bool is_cat
string image_path